js/player_thread.js

```diff
@@ -334,7 +334,12 @@
             });
             self.currentAudio.disabled = true;
             self.sendPlayingEvent('err');
-            self.currentHowl.unload();
+            for (let i = 0; i < self.playlist.length; i += 1) {
+              if (self.playlist[i].howl === self.currentHowl) {
+                self.playlist[i].howl = null
+              }
+            };
```

Comment:
```suggestion
            for (const i of self.playlist) {
              if (i.howl === self.currentHowl) {
                i.howl = null;
              }
            }
```
In favor of for...of syntax